BlueprintUICommonControls/Sources/UserInterfaceOverridingElement.swift

```diff
@@ -0,0 +1,73 @@
+import BlueprintUI
+import UIKit
+
+/// An element that allows overriding the user interface style (light/dark mode) for its wrapped content.
+///
+/// Use this element to force a specific appearance for a portion of your UI, regardless of the system-wide
+/// settings. This can be useful when you need to ensure certain UI elements maintain a consistent appearance
+/// across different user interface styles.
+///
+/// Example:
+/// ```swift
+/// let content = Label(text: "Hello, World!")
+/// let forcedLightMode = UserInterfaceOverridingElement(
+///     userInterfaceStyle: .light,
+///     wrapping: content
+/// )
+/// ```
+public struct UserInterfaceOverridingElement: Element {
```

Comment:
Yes, I think this is much more extensible! Couple thoughts:

1) We dropped the "style" from the name here, how about `OverrideUserInterfaceStyle`?

2) Can we make the element in the init be a trailing closure so you can do

```swift
OverrideUserInterfaceStyle(.light) {
  ... some element ...
}
```
?